haqq/chain.json

```diff
@@ -0,0 +1,187 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "haqq",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://haqq.network/",
+  "pretty_name": "Haqq Network",
+  "chain_id": "haqq_11235-1",
+  "bech32_prefix": "haqq",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1576 ✅